validation-test/stdlib/IOKitOverlay.swift

```diff
@@ -0,0 +1,31 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+// REQUIRES: OS=macosx
+
+import IOKit
+import IOKit.hid
+import StdlibUnittest
+
+// Also import modules which are used by StdlibUnittest internally. This
+// workaround is needed to link all required libraries in case we compile
+// StdlibUnittest with -sil-serialize-all.
+import SwiftPrivate
+#if _runtime(_ObjC)
```

Comment:
Since this only works on OS X, you can drop the `#if` around `import ObjectiveC`.